src/UglyToad.PdfPig/Content/Line.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UglyToad.PdfPig.Geometry;
+using System.Linq;
+
+namespace UglyToad.PdfPig.Content
+{
+    /// <summary>
+    /// A line.
+    /// </summary>
+    public class Line
```

Comment:
My worry with the name `Line` is that people might confuse it with the `PdfLine` / expect it to be a geometrical feature. Maybe we could use the name `TextLine` or something similar and make the class comment slightly more descriptive, e.g. a line of text in the document, or something.

Reply:
Makes sense, will adjust that